pkg/controller/atlasstream/atlasstream_connection.go

```diff
@@ -0,0 +1,256 @@
+package atlasstream
```

Comment:
Cannot see the main reconciler entry point nor the state machine definition or code.